### PR TITLE
updated webpack

### PR DESCRIPTION
**Pull request by NgoMike**

### CODE REVIEW

**Review by Busyreadingsomething — APPROVED** [review]
Looks good. Nice clean up on the vars to const

File: public/app-server.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...!==i||(e=null);var a=!1;if(null===e)a=!0;else switch(i){case"string":case"number":a=!0;break;case"object":switch(e.$$typeof){case T:case E:case C:case k:a=!0}}if(a)return n(o,e,""===t?"."+h(e,0):t),1;if(a=0,t=""===t?".":t+":",Array.isArray(e))for(var s=0;s<e.length;s++){i=e[s];var u=t+h(i,s);a+=d(i,u,n,o)}else if(null===e||void 0===e?u=null:(u=N&&e[N]||e["@@iterator"],u="function"==typeof u?u:null),"function"==typeof u)for(e=u.call(e),s=0;!(i=e.next()).done;)i=i.value,u=t+h(i,s++),a+=d(i,u,n,o);else"object"===i&&(n=""+e,r("31","[object Object]"===n?"object with keys {"+Object.keys(e).join(", ")+"}":n,""));return a}function h(e,t){return"object"==typeof e&&null!==e&&null!=e.key?c(e.key):t.toString(36)}function m(e,t){e.func.call(e.context,t,e.count++)}function g(e,t,n){var r=e.result,o=e.keyPrefix;e=e.func.call(e.context,t,e.count++),Array.isArray(e)?y(e,r,n,x.thatReturnsArgument):null!=e&&(l(e)&&(t=o+(!e.key||t&&t.key===e.key?"":(""+e.key).replace(P,"$&/")+"/")+n,e={$$typeof:T,type:e.type,key:t,ref:e.ref,props:e.props,_owner:e._owner}),r.push(e))}function y(e,t,n,r,o){var i="";null!=n&&(i=(""+n).replace(P,"$&/")+"/"),t=f(t,i,r,o),null==e||d(e,"",g,t),p(t)}/** @license React v16.2.0
```
Comment (MichaelDurfey) [inline public/app-server.js:1]: I'd gitignore this

File: public/app.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * react.production.min.js
```
Comment (MichaelDurfey) [inline public/app.js:2]: I'd git ignore this one too I believe

**Review by MichaelDurfey — APPROVED** [review]
Looks good. I'd git ignore your newrelic_agent.log and the two react files (i think that's what they are)

**Pull request merged**